docs/hugo/content/design/ADR-2023-07-Goal-Seeking-KeyVaults/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 2023-07 Goal Seeking KeyVaults
+---
+
+## Context
+
+Azure KeyVaults are soft deleted, to protect against accidental deletion. If you delete a KeyVault, you can't create a new KeyVault with the same name unless you explicit purge or recover the KeyVault.
```

Comment:
Also, a link to the official documentation would be good: https://learn.microsoft.com/en-us/azure/key-vault/general/soft-delete-overview.

Reply:
Or would be good to mention "The ability to opt out of soft-delete is deprecated and will be removed in February 2025"

Reply:
Done.